Cheerify/AddView.swift

```diff
@@ -117,6 +124,50 @@ struct AddView: View {
     }
 }
 
+struct PopUpView: View {
```

Comment:
nit: This should be added to a new separate view class to conform to MVVM architecture. I'll likely be going in and modifying file structure though, so not a huge deal

Reply:
sure 👍